bodhi/tests/server/consumers/test_greenwave.py

```diff
@@ -25,14 +25,14 @@
 from bodhi.server.consumers import greenwave
 from bodhi.server.config import config
 from bodhi.tests.server import create_update
-from bodhi.tests.server.base import BaseTestCase, TransactionalSessionMaker
+from bodhi.tests.server.base import BasePyTestCase, TransactionalSessionMaker
 
 
-class TestGreenwaveHandler(BaseTestCase):
+class TestGreenwaveHandler(BasePyTestCase):
     """Test class for the :func:`GreenwaveHandler` method."""
 
-    def setUp(self):
-        super().setUp()
+    def setup_method(self, method):
+        super(TestGreenwaveHandler, self).setup_method(method)
```

Comment:
FYI, Bodhi is needs at least Python 3, so you could also use `super().setup_method(method)` here.

Reply:
Thx for tip :)